Alarms.h

```diff
@@ -319,13 +335,24 @@ class AlarmManager {
   int led_pin_;
   utils::Pulse led_pulse_;
   Alarm alarms_[NUM_ALARMS];
+  Alarm alarmsHeader_[NUM_ALARMS-4];
+  Alarm alarmsFooter_[4];
   unsigned long const* cycle_count_;
 
   // Get number of alarms that are ON
   int numON() const;
 
-  // Get text to display
-  String getText() const;
+  // Get number of knob confirm alarms that are ON
+  int numON_Confirm() const;
+
+  // Get number of nonconfirm alarms that are ON
+  int numON_NonConfirm() const;
+
+  // Get header text to display
+  String getHeaderText() const;
+
+  // Get footer text to display
+  String getFooterText() const;
```

Comment:
The concept of header or footer doesn't make sense in the context of an alarm manager object. The AlarmManager class is meant as exactly that, a manager for alarms.

Reply:
The alarm manager now only retrieves the general alarm messages and the unconfirmed knob alerts, and passes those separately to the display. The concept of writing to a header vs footer is left to the display.